pkg/reconciler/revision/resources/deploy.go

```diff
@@ -50,6 +50,28 @@ var (
 		SubPathExpr: "$(K_INTERNAL_POD_NAMESPACE)_$(K_INTERNAL_POD_NAME)_",
 	}
 
+	varProjectedVolume = corev1.Volume{
+		Name: "knative-proj-volume",
+		VolumeSource: corev1.VolumeSource{
+			Projected: &corev1.ProjectedVolumeSource{
+				Sources: []corev1.VolumeProjection{
+					{
+						ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+							ExpirationSeconds: ptr.Int64(600),
+							Path:              "token",
+							Audience:          "freezer",
+						},
+					},
+				},
+			},
+		},
+	}
+
+	varProjectedVolumeMount = corev1.VolumeMount{
+		Name:      varProjectedVolume.Name,
+		MountPath: "/var/run/secrets/tokens",
```

Comment:
I assume the queue-proxy will pick this location up. Should it be defined as a constant somewhere?

Reply:
Yes, a constant makes sense. (Queue-proxy will pick up the token and send it to the freeze-proxy service as part of the freezing/thawing when scaling to/from zero.)

I think the best place to define one would be `pkg/queue/concurrency_state.go`. That file was created in #11783 but hasn't merged yet, so I'll update to a constant when I implement the use of the token in queue-proxy if that's ok (?).

Reply:
WFM!